src/transformers/models/wav2vec2/feature_extraction_wav2vec2.py

```diff
@@ -79,13 +79,20 @@ def __init__(
         self.do_normalize = do_normalize
 
     @staticmethod
-    def zero_mean_unit_var_norm(input_values: List[np.ndarray], input_lengths: List[int]) -> List[np.ndarray]:
+    def zero_mean_unit_var_norm(
+        input_values: List[np.ndarray], attention_mask: List[np.ndarray], padding_value=0.0
```

Comment:
seems `padding_value` is not used in this method

Reply:
It should be used ;-)